src/common/socket.c

```diff
@@ -238,7 +238,11 @@ fd_set readfds;
 
 // Maximum packet size in bytes, which the client is able to handle.
 // Larger packets cause a buffer overflow and stack corruption.
-static size_t socket_max_client_packet = 24576;
+#if PACKETVER >= 20131223
+static size_t socket_max_client_packet = 0x10000;
```

Comment:
So your going to set the buffer higher then the max the client can support? I wouldn't do it. When Ai4rei made this config in eA, she set it to 20480 which im guessing she has the source setup to reject anything larger. I just assumed if she put the exact value of what the old clients used, then using the exact value of what the new ones use would be safe. Unless dev's in Herc changed something on this.